### PR TITLE
Update UPGRADE.md

### PR DESCRIPTION
**Pull request by AsherThomasBabu**
Added the Breaking Changes missing for Upgrade to 0.14.x

### Summary

SUMMARY_GOES_HERE

### Full changelog

* [Implement ...]
* [Add related tests]
* ...

### Issues resolved

Fix #XXX

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -1093,6 +1093,14 @@ complete list of changes and new features.
   it if you are using a custom template. The changes are detailed in a diff
   included below.
 
+#### Plugins
+- The Runscope plugin has been dropped, based on the EoL announcement made by Runscope about their Traffic Inspector product. #3495
+
+#### Admin API
+- The SSL Certificates and SNI entities have moved to the new DAO implementation. As such, the /certificates and /snis endpoints have received notable usability improvements, but suffer from a few breaking changes. #3386
+
+- The Consumers entity has moved to the new DAO implementation. As such, the /consumers endpoint has received notable usability improvements, but suffers from a few breaking changes. #3437
```
Comment (hishamhm) [inline UPGRADE.md]:
```suggestion
- The Consumers entity has moved to the new DAO implementation. As such, the `/consumers` endpoint has received notable usability improvements, but suffers from a few breaking changes. [#3437](https://github.com/Kong/kong/pull/3437)
```

File: UPGRADE.md [hunk 2]
```diff
@@ -1093,6 +1093,14 @@ complete list of changes and new features.
   it if you are using a custom template. The changes are detailed in a diff
   included below.
 
+#### Plugins
+- The Runscope plugin has been dropped, based on the EoL announcement made by Runscope about their Traffic Inspector product. #3495
+
+#### Admin API
+- The SSL Certificates and SNI entities have moved to the new DAO implementation. As such, the /certificates and /snis endpoints have received notable usability improvements, but suffer from a few breaking changes. #3386
```
Comment (hishamhm) [inline UPGRADE.md]:
```suggestion
- The SSL Certificates and SNI entities have moved to the new DAO implementation. As such, the /certificates and /snis endpoints have received notable usability improvements, but suffer from a few breaking changes. [#3386](https://github.com/Kong/kong/pull/3386)
```

File: UPGRADE.md [hunk 3]
```diff
@@ -1093,6 +1093,14 @@ complete list of changes and new features.
   it if you are using a custom template. The changes are detailed in a diff
   included below.
 
+#### Plugins
+- The Runscope plugin has been dropped, based on the EoL announcement made by Runscope about their Traffic Inspector product. #3495
```
Comment (hishamhm) [inline UPGRADE.md]:
```suggestion
- The Runscope plugin has been dropped, based on the EoL announcement made by Runscope about their Traffic Inspector product. [#3495](https://github.com/Kong/kong/pull/3495)
```

File: UPGRADE.md [hunk 4]
```diff
@@ -1093,6 +1093,14 @@ complete list of changes and new features.
   it if you are using a custom template. The changes are detailed in a diff
   included below.
 
+#### Plugins
```
Comment (hishamhm) [inline UPGRADE.md]:
```suggestion
##### Plugins
```

File: UPGRADE.md [hunk 5]
```diff
@@ -1093,6 +1093,14 @@ complete list of changes and new features.
...
+- The Runscope plugin has been dropped, based on the EoL announcement made by Runscope about their Traffic Inspector product. #3495
+
+#### Admin API
```
Comment (hishamhm) [inline UPGRADE.md]:
```suggestion
##### Admin API
```

**Comment by AsherThomasBabu** [comment]
All the suggested changes have been applied! 
Thanks

**Comment by hishamhm** [comment]
@AsherThomasBabu thank you!

**Comment by AsherThomasBabu** [comment]
My pleasure @hishamhm !

**Pull request merged**